.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,7 @@
+contact_links:
+  - name: 📃 Bugs in React
+    url: https://github.com/facebook/react.dev/issues/new
+    about: This issue tracker is not for bugs in React. Please file React issues here.
```

seems contradictory? 
```suggestion
    about: This issue tracker is not for bugs in React. Please file React docs issues here.
```